src/utils/mesh_utils.ts

```diff
@@ -92,22 +95,20 @@ export const meshUtils = {
                 case OrderEventEndState.FullyFilled:
                 case OrderEventEndState.StoppedWatching:
                 case OrderEventEndState.Unfunded: {
-                    removed.push(apiOrder);
-                    persistentUpdated.push(apiOrder);
+                    removed.push(order);
                     break;
                 }
                 case OrderEventEndState.Unexpired:
                 case OrderEventEndState.FillabilityIncreased:
                 case OrderEventEndState.Filled: {
-                    updated.push(apiOrder);
-                    persistentUpdated.push(apiOrder);
```

Comment:
Instead of including persistent orders here, which Mesh doesn't know about, we'll handle the db update logic exclusively in OrderWatcherService.